pipelines/graphene_nightly_pipeline

```diff
@@ -79,30 +79,30 @@ pipeline {
+                        build job: 'local_ci_graphene_native_22.04_6.2', parameters: [[$class: 'StringParameterValue', name: 'stress_ng_run', value: 'True'], [$class: 'StringParameterValue', name: 'gramine_repo', value: gramine_repo], [$class: 'StringParameterValue', name: 'gramine_branch', value: gramine_branch]], propagate: true, wait: true
                     }
                 }
 
-                stage ('Invoke Ubuntu 22.04 with 6.0.9 Graphene SGX Suite') {
+                stage ('Invoke Ubuntu 22.04 with 6.2 Graphene SGX Suite') {
                     steps {
                         sleep(time: 15, unit: "MINUTES")
-                        build job: 'local_ci_graphene_sgx_22.04_6.0.9', parameters: [[$class: 'StringParameterValue', name: 'stress_ng_run', value: 'True'], [$class: 'StringParameterValue', name: 'gramine_repo', value: gramine_repo], [$class: 'StringParameterValue', name: 'gramine_branch', value: gramine_branch]], propagate: true, wait: true
+                        build job: 'local_ci_graphene_sgx_22.04_6.2', parameters: [[$class: 'StringParameterValue', name: 'stress_ng_run', value: 'True'], [$class: 'StringParameterValue', name: 'gramine_repo', value: gramine_repo], [$class: 'StringParameterValue', name: 'gramine_branch', value: gramine_branch]], propagate: true, wait: true
                     }
                 }
 
-                stage ('Invoke CentOS 8 with 6.0 Graphene Native Suite ') {
+                stage ('Invoke CentOS 8 with 6.2.8 Graphene Native Suite ') {
                     steps {
                         sleep(time: 30, unit: "MINUTES")
-                        build job: 'local_ci_graphene_native_centos_6.0', parameters: [[$class: 'StringParameterValue', name: 'gramine_repo', value: gramine_repo], [$class: 'StringParameterValue', name: 'gramine_branch', value: gramine_branch]], propagate: true, wait: true
+                        build job: 'local_ci_graphene_native_centos_6.2.8', parameters: [[$class: 'StringParameterValue', name: 'gramine_repo', value: gramine_repo], [$class: 'StringParameterValue', name: 'gramine_branch', value: gramine_branch]], propagate: true, wait: true
                     }
                 }
 
-                stage ('Invoke CentOS 8 with 6.0 Graphene SGX Suite ') {
+                stage ('Invoke CentOS 8 with 6.2.8 Graphene SGX Suite ') {
                     steps {
                         sleep(time: 45, unit: "MINUTES")
-                        build job: 'local_ci_graphene_sgx_centos_6.0', parameters: [[$class: 'StringParameterValue', name: 'gramine_repo', value: gramine_repo], [$class: 'StringParameterValue', name: 'gramine_branch', value: gramine_branch]], propagate: true, wait: true
```

Comment:
Now, that we have 2 jobs for centos, can you rename this and native job to local_ci_graphene_sgx_centos_8_6.2